matlab/test/arrow/array/tListArray.m

```diff
@@ -160,6 +184,81 @@ function TestErrorIfEmptyOffsets(testCase)
             testCase.verifyError(fcn, "arrow:array:ListArrayFromArraysFailed");
         end
 
+        function TestValidationModeDefault(testCase, TestValidationModeArray)
+            % Verify that the default ValidationMode value for the
+            % arrow.array.ListArray.fromArrays method is
+            % arrow.array.ValidationMode.Minimal.
+            offsets = TestValidationModeArray.Offsets;
+            values = TestValidationModeArray.Values;
+            valid = TestValidationModeArray.Valid;
+            fcn = @() arrow.array.ListArray.fromArrays(offsets, values);
+            if ~valid
```

Comment:
You could filter this test if `TestValidatorModeArray.Valid` is `true`:

```matlab
tc.assumeFalse(TestValidatorModeArray.Valid, "Only run with invalid arrays");
```

Reply:
Perhaps there is a misunderstanding.

I don't want to filter these test points because I am intentionally testing that a valid array with a `ValidationMode` that is not `"none"` should not throw an error.

Reply:
Oh, sorry, I made a mistake.

I forgot to add the `else` clause here to be:

```matlab
if ~valid
   testCase.verifyError(fcn, "arrow:array:ValidateMinimalFailed");
else
   testCase.verifyWarningFree(fcn);
end
```
Thanks for catching this. I'll update the tests.

Reply:
Fixed.